Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -150,33 +110,57 @@ extension FacilitiesDropdownCell: UICollectionViewDataSource {
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        // TODO: once facility cells are finished
-        return UICollectionViewCell(frame: .zero)
+        switch facility.details[indexPath.row].detailType {
+        case .equipment:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilityEquipmentListCell, for: indexPath) as! EquipmentListCell
+            cell.configure(for: facility.details[indexPath.row].getEquipmentCategories())
+            return cell
+        case .hours:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilityHoursCell, for: indexPath) as! FacilitiesHoursCell
+            cell.configure(facilityDetail: facility.details[indexPath.row], dayIndex: 0, onChangeDay: nil)
```

Comment:
`onChangeDay` is `nil`? does the height of the cell change correctly when you switch the day in the calendar/hours view?